### PR TITLE
Works in bav114-118

### PR DESCRIPTION
**Pull request by MassimoVilla**
Following [#2684](https://github.com/BetaMasaheft/Documentation/issues/2684) I have created three new work records and corrected the incipit of an existing record

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]
thank you!

File: new/LIT7138PrLover.xml
Comment (nafisa-valieva) [inline new/LIT7138PrLover.xml:1]: Massimo, I think it would be nice to have an abstract of the text. We think it would be a nice practice in general.

**Review by nafisa-valieva — APPROVED** [review]

File: new/LIT7138PrLover.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7138PrLover" passive="BAVet118#a1"/>
+                    <relation name="lawd:hasAttestation" active="LIT7138PrLover" passive="BAVet128#a33"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>
+                        The following incipit is taken from <ref type="mss" corresp="BAVet118"/> according to
+                        <bibl><ptr target="bm:GrebTiss1935Codices"/><citedRange unit="page"></citedRange></bibl>.</note>
```
Comment (karljonaskarlsson) [inline new/LIT7138PrLover.xml:67]: Is the page number missing?

File: new/LIT7138PrLover.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7138PrLover" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Prayer of the lover</title>
+                <title xml:lang="gez" xml:id="t2">ጸሎተ፡ መስተፋቅር</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Ṣalota mastafāqər</title>
+                <title xml:lang="grc" xml:id="t3">περη αγαπης κὲ εφτήχία</title>
```
Comment (karljonaskarlsson) [inline new/LIT7138PrLover.xml:11]: I agree that we really need an abstract to explain that this Greek "title" comes from the manuscript itself and that the spelling has not been standardized.

The catalogue seems to have εφτηχία, with only one accent

File: new/LIT7138PrLover.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                    <relation name="lawd:hasAttestation" active="LIT7138PrLover" passive="BAVet128#a33"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>
+                        The following incipit is taken from <ref type="mss" corresp="BAVet118"/> according to
+                        <bibl><ptr target="bm:GrebTiss1935Codices"/><citedRange unit="page"></citedRange></bibl>.</note>
+                    <ab>
+                        በስመል፡ አብ፡ <gap reason="ellipsis" resp="PRS4805Grebaut PRS9530Tisseran"/> <sic>ጸሎተ፡</sic> <sic>ምስተ<surplus>፡</surplus>ፋቅር፨</sic>
```
Comment (karljonaskarlsson) [inline new/LIT7138PrLover.xml:69]: በስመ in the catalogue

**Review by karljonaskarlsson — APPROVED** [review]

**Review by nafisa-valieva — APPROVED** [review]
Thank you!

**Pull request merged**